gatsby/content/projects/bots/bucktrix.mdx

```diff
@@ -0,0 +1,32 @@
+---
+layout: projectimage
+title: finzzz / bucktrix
+categories:
+- bot
+description: Extensible matrix bot
+author: finzzz
+repo: https://github.com/finzzz/bucktrix
+language: Python3
+license: GPLv2
+thumbnail: /docs/projects/images/bucktrix.svg
+featured: true
+maturity: Alpha
+---
+
+Visit repo for more information.
+
+## What is Bucktrix
+Bucktrix is a personal [matrix](https://github.com/matrix-org/synapse) chatbot that can send message or execute commands based on certain events.
```

Comment:
```suggestion
Bucktrix is a personal Matrix chatbot that can send message or execute commands based on certain events.
```

The Matrix link doesn't seem as relevant here when you're already on Matrix.org. I see that this is the same text from the readme where it makes more sense.